account_check_deposit_operating_unit/models/check_deposit.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class AccountCheckDeposit(models.Model):
+    _inherit = "account.check.deposit"
+
+    operating_unit_id = fields.Many2one(
```

Comment:
I think you should have index here.
Without it, using security rule when we have big number of records will be slow.

Reply:
Yes, I agree with you.

Edited.